bin/run-tests

```diff
@@ -16,7 +16,7 @@ docker exec test-backend bash -c "yarn db:seed;"
 docker exec test-backend bash -c "yarn test:ci"
 
 # Test frontend
-docker exec test-backend bash -c "yarn --cwd frontend run test:ci"
+docker exec test-frontend bash -c "yarn --cwd frontend run test:ci"
```

Comment:
This doesn't work. Looks like a missing `container_name` for the frontend service. `--cwd frontend` is also unnecessary (though doesn't break anything) in the frontend container.

<img width="452" alt="Screen Shot 2021-02-05 at 9 35 36 AM" src="https://user-images.githubusercontent.com/285842/107047180-85f2c500-6795-11eb-960f-5f3f87161950.png">

That also exposes that this script papers over any failures if you don't read the output carefully. Since this is not used in CI/CD I think I'm ok with that, but I think it'd be good to collect exit statuses and exit the script with non-zero if any individual step exists with non-zero.

Reply:
Hmmm, I've run this a few times and haven't seen this problem. I'll take a look again. Could you get me the output of `docker-compose -f docker-compose.test.yml ps --all` on your machine?

As for handing failures, it was intentional that we didn't error out if one of the intermediate steps threw an error. We could catch those statuses, but I'm not sure we want to prevent frontend tests from running if the backend tests fail, for example.

Reply:
> I'm not sure we want to prevent frontend tests from running if the backend tests fail, for example

Yeah, I went back and forth on that. I'm 100% fine with all the steps running as long as I get a nice clear indication that at least one step failed when the script exits.

Here's the `ps` output:

<img width="701" alt="Screen Shot 2021-02-05 at 10 20 14 AM" src="https://user-images.githubusercontent.com/285842/107052708-f866a380-679b-11eb-896b-c3366d49d48c.png">

Reply:
Just a quick check of the doc page for [`docker-compose exec`](https://docs.docker.com/compose/reference/run/), it sounds like that command takes a _service_ name rather than a _container_ name, so that might be related.

While we're discussing, I had originally considered having the frontend and backend tests run as separate scripts, since  it seems like engineers might be focused on either frontend or backend during development. Would you be interested in making this accept an optional frontend/backend param?

Reply:
> Would you be interested in making this accept an optional frontend/backend param

If it's quick, sure, but that's pretty low priority to me.

Reply:
It was quick: https://github.com/adhocteam/Head-Start-TTADP/pull/149/files

I also added some logging of errors and have the script return an exit code value equal to the number of script failures.